profile/README.md

```diff
@@ -1,11 +1,19 @@
 ## AICA Technology
 
-### Repositories
+This the home of the AICA robotics framework!
```

Comment:
Is the missing 'is' intentional? I would still approve it :smile:

Reply:
haha thanks for spotting that